common/component/redis/redis.go

```diff
@@ -15,15 +15,24 @@ package redis
 
 import (
 	"context"
+	"errors"
 	"fmt"
 	"strconv"
 	"strings"
 	"time"
 
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/policy"
+	"github.com/cenkalti/backoff/v4"
+	"github.com/golang-jwt/jwt/v5"
```

Comment:
In the past we tried to standardize on using [github.com/lestrrat-go/jwx/v2/jwt](https://pkg.go.dev/github.com/lestrrat-go/jwx/v2/jwt) for all jw* things, to avoid adding (direct) dependencies on other packages that do similar things. Please consider switching to this library

Reply:
We should probably make that a linter rule then! :)